### ISSUE TITLE
NPE when tapping on ETA in header

### ISSUE THREAD

**Issue by barbeau** [OneBusAway/onebusaway-android#297]
For `develop` branch:

On Samsung Galaxy S3 with Android 4.4.4 (CyanogenMod):

~~~
    Process: com.joulespersecond.seattlebusbot, PID: 8561
    java.lang.NullPointerException
            at android.widget.TextView.checkForRelayout(TextView.java:6556)
            at android.widget.TextView.setText(TextView.java:3813)
            at android.widget.TextView.setText(TextView.java:3671)
            at android.widget.TextView.setText(TextView.java:3646)
            at org.onebusaway.android.ui.ArrivalsListHeader.setupPopup(ArrivalsListHeader.java:755)
            at org.onebusaway.android.ui.ArrivalsListHeader.access$600(ArrivalsListHeader.java:66)
            at org.onebusaway.android.ui.ArrivalsListHeader$10.onClick(ArrivalsListHeader.java:689)
            at android.view.View.performClick(View.java:4445)
            at android.view.View$PerformClick.run(View.java:18446)
            at android.os.Handler.handleCallback(Handler.java:733)
            at android.os.Handler.dispatchMessage(Handler.java:95)
            at android.os.Looper.loop(Looper.java:136)
            at android.app.ActivityThread.main(ActivityThread.java:5146)
            at java.lang.reflect.Method.invokeNative(Native Method)
            at java.lang.reflect.Method.invoke(Method.java:515)
            at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:732)
            at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:566)
            at dalvik.system.NativeStart.main(Native Method)
~~~

**Comment by barbeau** [comment]
Same crash on Amazon Fire Phone w/ Android 4.4.4:

~~~
java.lang.NullPointerException
            at android.widget.TextView.checkForRelayout(TextView.java:6659)
            at android.widget.TextView.setText(TextView.java:3894)
            at android.widget.TextView.setText(TextView.java:3752)
            at android.widget.TextView.setText(TextView.java:3727)
            at org.onebusaway.android.ui.ArrivalsListHeader.setupPopup(ArrivalsListHeader.java:755)
            at org.onebusaway.android.ui.ArrivalsListHeader.access$600(ArrivalsListHeader.java:66)
            at org.onebusaway.android.ui.ArrivalsListHeader$9.onClick(ArrivalsListHeader.java:637)
            at android.view.View.performClick(View.java:4768)
            at android.view.View$PerformClick.run(View.java:19073)
            at android.os.Handler.handleCallback(Handler.java:755)
            at android.os.Handler.dispatchMessage(Handler.java:95)
            at android.os.Looper.loop(Looper.java:145)
            at android.app.ActivityThread.main(ActivityThread.java:5266)
            at java.lang.reflect.Method.invokeNative(Native Method)
            at java.lang.reflect.Method.invoke(Method.java:515)
            at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:826)
            at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:642)
            at dalvik.system.NativeStart.main(Native Method)
~~~

**Comment by barbeau** [comment]
Looks like this is a bug in Android that affects Android 4.4.3 and 4.4.4:
https://code.google.com/p/android/issues/detail?id=75877

It seems to be fixed in Android 5.1, as it doesn't crash on an LG G4 with Android 5.1.

There is a workaround, which is to set the layout parameters in code:

`statusView.setLayoutParams(new ViewGroup.LayoutParams(ViewGroup.LayoutParams.WRAP_CONTENT, ViewGroup.LayoutParams.WRAP_CONTENT));`

See also:
http://stackoverflow.com/a/15862282/937715

**Comment by barbeau** [comment]
Fixed in `develop` via https://github.com/OneBusAway/onebusaway-android/commit/f1bd35be8445dfab7bd73c124eaadb88c7143dbe.
